src/main/java/com/google/neighborgood/servlets/TaskServlet.java

```diff
@@ -93,8 +96,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       }
       out.append("<div class='task-container'>");
       out.append("<div class='task-header'>");
+
+      // Retrieves task's user to get user's nickname
+      String taskUserId = (String) entity.getProperty("userId");
+      Query userQuery =
+          new Query("UserInfo")
+              .setFilter(new FilterPredicate("userId", FilterOperator.EQUAL, taskUserId));
+      PreparedQuery userResults = datastore.prepare(userQuery);
+      Entity userEntity = userResults.asSingleEntity();
+
```

Comment:
nit: I would recommend to add an error handling after line 106, checking whether `userEntity` is `null`. If it's null, we should return a 404 error message. You could refer to the `doPost()` function in `TaskInfoServlet.java` file.

Reply:
Good idea! I’ll add this in the next commit, thanks!